response_operations_ui/static/scss/components/table.scss

```diff
@@ -0,0 +1,43 @@
+table {
```

Comment:
Has this way of dealing with the css been agreed with @nicprice?

Reply:
Meeting with Nic later today :+1:

Reply:
@tej99 Confirmed current pattern with Nic